docs/api/schema.md

```diff
@@ -18,23 +18,23 @@ While the default `_id` property is added with an `ObjectId` type, its type can
 
 **Parameters:**
 
-| Name | Type | Attribute |
-| --- | --- | --- |
-| `properties` | `Record<string, unknown>` | required |
-| `options` | `SchemaOptions` | optional |
-| `options.defaults` | `Partial<TProperties>` | optional |
-| `options.timestamps` | `boolean` | optional |
-| `options.validationAction` | `VALIDATION_ACTION` | optional |
-| `options.validationLevel` | `VALIDATION_LEVEL` | optional |
+| Name                       | Type                      | Attribute |
```

Comment:
These edits look like they were manual. These files are generated automatically when running `yarn docs`. Please run that instead and commit the resulting files.

Reply:
So I just did that and now the bad line endings are back :/